prboom2/src/m_menu.c

```diff
@@ -3949,7 +3950,6 @@ setup_menu_t helpstrings[] =  // HELP screen strings
   {"STRAFE RIGHT",S_SKIP|S_INPUT,m_null,KT_X3,0,dsda_input_straferight},
   {"STRAFE"      ,S_SKIP|S_INPUT,m_null,KT_X3,0,dsda_input_strafe},
   {"AUTORUN"     ,S_SKIP|S_INPUT,m_null,KT_X3,0,dsda_input_autorun},
-  {"180 TURN"    ,S_SKIP|S_INPUT,m_null,KT_X3,0,dsda_input_reverse},
```

Comment:
Why is this change here?

Reply:
Just something I stumbled upon while working with the sources. If there's a reason it should not be removed, then you can drop this commit (idk if that's how it works)